.github/workflows/release.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.1.1
```

Comment:
We probably should bump this to at least >=2.1.3 if we're built 3.10. Cibuildwheel was using beta and release candidates for py3.10 on versions <2.1.3